lmdeploy/turbomind/deploy/target_model/bf16.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from lmdeploy.turbomind.deploy.source_model.base import BaseInputModel
+
+from .base import OUTPUT_MODELS, TurbomindModelConfig
+from .fp import TurbomindModel
+
+
+@OUTPUT_MODELS.register_module(name='bf16')
+class TurbomindBF16Model(TurbomindModel):
```

Comment:
Is it a must? Can't fp.py cover it?

Reply:
I think it is not necessary to add another class `TurbomindBF16Model`. We can use this way:
```
@OUTPUT_MODELS.register_module(name=['fp16', 'bf16'])
class TurboMindModel:
 ...
```